app/routes/events/components/EventAdministrate/Administrate.css

```diff
@@ -24,33 +24,72 @@
   grid-template-columns: repeat(5, 20%);
 }
 
-.webkomIcon {
-  font-size: 1.1em;
-  background: #e20d13;
-  color: var(--color-white);
+.pill {
+  margin: 0 auto;
   display: inline-block;
-  border-radius: 60px;
-  padding: 0.1em 0.2em;
+  text-transform: uppercase;
+  padding: 0.25rem 0.375rem;
+  margin: 0px 2px;
+  font-size: 0.7rem;
+  font-weight: bold;
+  border-radius: 1rem;
+  letter-spacing: 0.6px;
+  line-height: 1;
+  box-shadow: rgb(0 0 0 / 5%) 1px 2px 5px 0px;
+  align-items: center;
+  align-self: center;
+}
+
+.webkomPill {
```

Comment:
:eyes:

Reply:
Maybe move the pill into another component? Also, now we have the pills used for tags and pills used here? What's the difference?

Reply:
Well, they are sort of different in the way you interact with them. I thought about "merging" them together though, but that will require more work, so I'll probably create a new PR.

Reply:
The tags will be affected by the color changes I'm going to make, so I'll do it then. I imagine next week.